django/contrib/contenttypes/fields.py

```diff
@@ -236,21 +218,29 @@ def __get__(self, instance, cls=None):
         if instance is None:
             return self
 
+        # Make sure to use ContentType.objects.get_for_id() to ensure that
```

Comment:
Exactly as @charettes writes – the comment merely explains why the following two lines of code fetch the `ForeignKey` field pointing to `ContentType` and get the value of its attname, instead of just triggering its descriptor to get the related `ContentType` object.